### PR TITLE
BRS:289 - Create FAQ page for DUP-Public

### PR DESCRIPTION
**Pull request by Christopher-walsh22**
### Jira Ticket:

BRS-289

### Jira Ticket URL:
[BRS-289](
https://github.com/orgs/bcgov/projects/49/views/1?pane=issue&itemId=43920696)

Also Includes: 
[BRS-302](
https://github.com/orgs/bcgov/projects/49/views/1?pane=issue&itemId=47055380)
### Description:

Implements the FAQ page for DUP Public
Updates the Links previously used to contact support with Link to new FAQ page.
Template for FAQ includes static information for contacting support along with admin set FAQ

### CODE REVIEW

File: src/app/services/faq.service.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      this.eventService.setError(
+        new EventObject(
+          EventKeywords.ERROR,
+          e,
+          'Pass Service'
+        )
+      );
+      throw e;
+    }
+    return res[0].text;
```
Comment (cameronpettit) [inline src/app/services/faq.service.ts:34]: return from within the try catch. This is currently bombing when the api fails to get the faq object

File: src/app/services/faq.service.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    private loggerService: LoggerService
+  ) {
+  }
+
+  async getFaq() {
+    let res = null;
+    try {
+      this.loggerService.debug(`GETTING FAQ: }`);
+      res = await this.apiService.get('readfaq', { faq: 'faq' });
+      console.log(JSON.stringify(res))
```
Comment (cameronpettit) [inline src/app/services/faq.service.ts:22]: remove console log

**Review by cameronpettit — CHANGES_REQUESTED** [review]

File: src/app/services/faq.service.ts (hunk 1 above)
Comment (Christopher-walsh22) [reply src/app/services/faq.service.ts]: Addressed in: [Commit](https://github.com/bcgov/parks-reso-public/pull/313/commits/d9e72091727546caaf8d450f67098c7d4ad47ede)

File: src/app/services/faq.service.ts (hunk 2 above)
Comment (Christopher-walsh22) [reply src/app/services/faq.service.ts]: Addressed in: [Commit](https://github.com/bcgov/parks-reso-public/pull/313/commits/d9e72091727546caaf8d450f67098c7d4ad47ede)

**Review by cameronpettit — APPROVED** [review]

**Pull request merged**